libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -77,13 +78,31 @@ export const CoreHeaderHelp: React.FC<IHeaderHelpProps> = ({
         onMenuItemClick(...args);
     };
 
+    /**
+     * Menu dropdown content is long enough to make it max-width (230px),
+     * so it should just switch alignment to bottom right corner of the Help button.
+     */
+    const getHelpDropdownAlignPoints = (): HelpMenuDropdownAlignPoints => {
```

Comment:
As the name of the function is `getHelpDropdownAlignPoints`, which ends in plural `AlignPoints`, it return value is supposed to be some array, isn't it?

Should we change the return type of the function to be an array instead?

```
const getHelpDropdownAlignPoints = (): HelpMenuDropdownAlignPoints => {
```

↓

```
const getHelpDropdownAlignPoints = (): HelpMenuDropdownAlignPoints[] => {
```

Reply:
My bad, as the type `HelpMenuDropdownAlignPoints` is already in plural, please ignore this comment.